src/interface/report/Results/TimelineTab.tsx

```diff
@@ -79,10 +86,9 @@ const TimelineTab = ({ parser }: Props) => {
             parser.fight.start_time,
             parser.fight.end_time,
             undefined,
-            "type = 'damage' AND source.type in ('NPC', 'Boss')",
+            "type = 'damage' AND source.type in ('NPC', 'Boss') AND ability.id > 1 AND source.id > 0 AND target.type = 'Player'",
```

Comment:
making a note for myself that this *could* fail if a boss/npc casts an ability which then has damage sourced to the environment (which does happen, unfortunately). but this should be fine in 99% of cases and deals with the outliers downloading several MB of JSON